### PR TITLE
Update ediscovery-premium-limits.md

### PR DESCRIPTION
**Pull request by StefanieBier**
Clear language

### CODE REVIEW

File: microsoft-365/compliance/ediscovery-premium-limits.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following table lists the limits for cases and review sets in eDiscovery (Pr
 |:---|:---|:---|
 |Total number of documents that can be added to a case (for all review sets in a case).|3 million|40 million|
 |Total file size per load set. This includes loading non-Office 365 into a review set.|300 GB|1 TB|
-|Maximum number of load sets per case.|200|200|
+|Maximum number of load sets (Collection jobs) per case.|200|200|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-premium-limits.md:39]:
```suggestion
|Maximum number of load sets (collection jobs) per case.|200|200|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @StefanieBier, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@StefanieBier Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@StefanieBier Thanks for your feedback! New issues and pull requests for this content are closed while we update this content to support upcoming feature improvements.

**Pull request closed without merging**